cmd/earthly/satellite_cmds.go

```diff
@@ -296,17 +296,23 @@ func (app *earthlyApp) actionSatelliteInspect(cliCtx *cli.Context) error {
 		app.buildkitdSettings.BuildkitAddress = containerutil.SatelliteAddress
 	}
 
-	err = buildkitd.PrintSatelliteInfo(cliCtx.Context, app.console, Version, app.buildkitdSettings)
-	if err != nil {
-		return errors.Wrap(err, "failed checking buildkit info")
-	}
-
 	selected := "No"
 	if selectedSatellite == satellite.Name {
 		selected = "Yes"
 	}
+
 	app.console.Printf("Instance state: %s", satellite.Status)
 	app.console.Printf("Currently selected: %s", selected)
+	app.console.Printf("")
+
+	if satellite.Status == cloud.SatelliteStatusOperational {
+		err = buildkitd.PrintSatelliteInfo(cliCtx.Context, app.console, Version, app.buildkitdSettings)
+		if err != nil {
+			return errors.Wrap(err, "failed checking buildkit info")
+		}
+	} else {
+		app.console.Printf("More info available when Satellite is awake.")
```

Comment:
In a follow up PR, I will add a command for `earthly satellite wake <my-sat>` , but I want to update that endpoint on the backend to use gRPC streams first.

Reply:
It's also worth mentioning that the state displayed above might now actually say "Asleep" in all cases. 

It could be something else, such as "Stopping", "Starting", or "Destroying" 

however, I think saying "...when Satellite is awake" is still clear enough.